recipes/gfaviz/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "gfaviz" %}
+{% set version="1.0.0" %}
+{% set hash="4bc4327dd182812a9cfd53a39632bf41f461170e142df62ebbd0ba441239a6d4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [ windows ]
```

Comment:
No need. Windows is not supported anyway

Reply:
I removed it from recipe